cli/core/pkg/cli/multirepo_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"context"
+)
+
+type FakeRepository struct {
+	Repository
+}
+
+func (f *FakeRepository) List() ([]Plugin, error) {
+	return []Plugin{}, context.DeadlineExceeded
+}
+
+// TODO: This is legacy code tests and will be removed soon. This test and implementation file would be removed soon
```

Comment:
There is an Github issue filed to remove this code https://github.com/vmware-tanzu/tanzu-framework/issues/3224.
Sine we are going to remove it soon, and as it requires to create a moc plugins,it has dependency on builder plugin which we want to avoid at the moment. So to avoid this, commented these tests.